### PR TITLE
Fix parser code for dataplugin which uses manifest to predefine parser

### PR DESCRIPTION
**Pull request by sengineer0**
https://github.com/biothings/biothings.api/issues/259

### CODE REVIEW

**Comment by newgene** [comment]
@sengineer0 Just to be sure, did you test that this fix works for the examples from this issue?

**Comment by sengineer0** [comment]
> @sengineer0 Just to be sure, did you test that this fix works for the examples from this issue?

@newgene  Yes, of course. I tested the fix against these data-plugin, and it worked well.

[https://github.com/ericz1803/doid](https://github.com/ericz1803/doid/blob/37c9bda7ba0e0569dad3181842ebc14d3af6c6a9/manifest.json#L16-L19)
[https://github.com/ericz1803/mondo](https://github.com/ericz1803/mondo/blob/73c33ba83f0a66329ede28d0dbcf7e80a00cd337/manifest.json#L16-L20)
https://github.com/remoteeng00/pharmgkb/tree/pharmgkb_v6

File: biothings/hub/dataplugin/assistant.py [hunk 1]
```diff
@@ -302,24 +302,35 @@ def get_uploader_dynamic_class(self, uploader_section, metadata, sub_source_name
...
                     try:
                         from {self.plugin_name}.{mod} import {func} as parser_func
                     except ImportError:
                         from .{mod} import {func} as parser_func
+
+                    parser_kwargs_serialized = r\'\'\'
+                        {parser_kwargs_serialized}
+                    \'\'\'  # I am not 100 percent certain this works
+
+                    parser_kwargs = orjson.loads(parser_kwargs_serialized)
```
Comment (newgene) [inline biothings/hub/dataplugin/assistant.py]: @sengineer0 is `repr` a better way to get the string representation of an object?

Comment (sengineer0) [reply biothings/hub/dataplugin/assistant.py]: not sure, let's me check it first

Comment (sengineer0) [reply biothings/hub/dataplugin/assistant.py]: @newgene  it's good idea. I tested and it worked well. I also updated and pushed the new change.
Please take a look, thanks :)

**Pull request merged**